.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
as with earlier comment add .vscode dir to .gitignore and remove this file.

Reply:
ok sir I will change it